native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarBatchScanExec.scala

```diff
@@ -25,8 +25,8 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.execution.metric.{SQLMetric, SQLMetrics}
 import org.apache.spark.sql.vectorized.{ColumnarBatch, ColumnVector}
 
-class ColumnarBatchScanExec(output: Seq[AttributeReference], @transient scan: Scan)
-    extends BatchScanExec(output, scan) {
+class ColumnarBatchScanExec(output: Seq[AttributeReference], @transient scan: Scan, runtimeFilters: Seq[Expression])
```

Comment:
Move this class into shim layers respectively for spark 3.1/3.2.
Code related to Gazelle config is removed to get rid of GazellePluginConfig dependency. If not, cyclic dependency needs to be handled.